ush/python/pygfs/task/aero_analysis.py

```diff
@@ -167,10 +165,12 @@ def finalize(self: Analysis) -> None:
         }
         FileHandler(yaml_copy).sync()
 
-        # ---- NOTE below is 'temporary', eventually we will not be using FMS RESTART formatted files
-        # ---- all of the rest of this method will need to be changed but requires model and JEDI changes
         # ---- copy RESTART fv_tracer files for future reference
-        template = '{}.fv_tracer.res.tile{}.nc'.format(to_fv3time(self.task_config.current_cycle), '{tilenum}')
+        if self.task_config.DOIAU:
+            restarttime = self.task_config.AERO_WINDOW_BEGIN
+        else:
+            restarttime = self.task_config.current_cycle
```

Comment:
Should this be elevated to `__init__` and compute as `model_start_time` once and for all?
Perhaps in `__init__` we can also compute `background_times` list once.
Thoughts?

Reply:
I think that's a good suggestion. Do we want that as part of this PR or separate?

Reply:
If you could move it to `__init__` in this PR itself, that would be great.